src/main/pg/rx.c

```diff
@@ -34,7 +34,7 @@
 #include "rx/rx.h"
 #include "rx/rx_spi.h"
 
-PG_REGISTER_WITH_RESET_FN(rxConfig_t, rxConfig, PG_RX_CONFIG, 2);
+PG_REGISTER_WITH_RESET_FN(rxConfig_t, rxConfig, PG_RX_CONFIG, 3);
```

Comment:
Why is this changed?

Reply:
Application of rule 6. The default value for rssi_sbus_lpf_period is 30, not zero, so I updated the version "in order to ensure the parameter is initialised to a valid value". Is that not a valid interpretation of rule 6?

Reply:
> 6. Any changes to a parameter group that are not covered in 5.

No, nada, nope, I can't see any change to the parameter group that goes beyond adding the element as covered in 5.

Reply:
OK, misunderstanding undone. :)